### ISSUE TITLE
setReqHeader parsing comma when it shouldn't

### ISSUE THREAD

**Issue by kungfu71186** [vfarcic/docker-flow-proxy#333]
Not sure if there is a way to escape a comma, but I'm attempting to do:

com.df.setReqHeader=ssl_client_cert -----BEGIN\ CERTIFICATE-----\ %[ssl_c_der,base64]\ -----END\ CERTIFICATE-----\  if { ssl_fc_has_crt }

It appears it tries to parse the comma and I get

http-request set-header ssl_client_cert -----BEGIN\ CERTIFICATE-----\ %!\(BADINDEX) -----END\ CERTIFICATE-----\  if { ssl_fc_has_crt }

#331 - This error does not validate from haproxy, but it tries to reload with the bad config anyway and crashes the proxy.

**Comment by kungfu71186** [comment]
@vfarcic This may want to be re-evaluated afterwards since HAproxy allows commas in a lot of fields. Maybe pick a different delimiter instead.

For now, using the csv parser might work ok. Using quotes around the line with a comma will exclude it from being split up.

```
package main
import (
	"fmt"
	"strings"
	"encoding/csv"
)

func main() {
	s := `"-----BEGIN\ CERTIFICATE-----\ %[ssl_c_der,base64]\ -----END\ CERTIFICATE-----\ if { ssl_fc_has_crt }"`
	r := csv.NewReader(strings.NewReader(s))
	r.LazyQuotes = true
	result, _ := r.ReadAll()
	
    	fmt.Printf("Tokens: %v", len(result[0]))
    	fmt.Println()
	
	fmt.Printf("%#v", result)
	fmt.Println()
	fmt.Println(result[0][0])
}

```


```
[][]string{[]string{"-----BEGIN\\ CERTIFICATE-----\\ %[ssl_c_der,base64]\\ -----END\\ CERTIFICATE-----\\ if { ssl_fc_has_crt }"}}
-----BEGIN\ CERTIFICATE-----\ %[ssl_c_der,base64]\ -----END\ CERTIFICATE-----\ if { ssl_fc_has_crt }
```

**Comment by vfarcic** [comment]
Available in the release [17.09.25-31](https://github.com/vfarcic/docker-flow-proxy/releases/tag/17.09.25-31).

Can you please try it out and let me know if you have a suggestion for a better implementation or if you experience a problem.

**Comment by kungfu71186** [comment]
@vfarcic looks good. Used a separator of "|" and was able to use a comma in the header. Thanks!
